pkg/pubsub/adapter/adapter.go

```diff
@@ -261,3 +245,15 @@ func (a *Adapter) obsNewHTTPClient(ctx context.Context, target string) (cloudeve
 	// Use the transport to make a new CloudEvents client.
 	return cloudevents.NewClient(t)
 }
+
+// resourceGroupFromEventType returns the resource group for the given eventType.
+func (a *Adapter) resourceGroupFromEventType(eventType string) string {
```

Comment:
thiz seems like it should be in the converter function area, not here.

Reply:
yeah, good point... let me move it there